cupertino_gallery/lib/widgets/alert_dialog_page.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/cupertino.dart';
+
+class AlertDialogPage extends StatelessWidget {
+  const AlertDialogPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      navigationBar: const CupertinoNavigationBar(
+        middle: Text('Alert Dialog'),
+      ),
+      child: Center(
+        child: CupertinoButton(
+          child: const Text('Show Alert Dialog'),
+          onPressed: () {
+            showCupertinoDialog<void>(
+              context: context,
+              builder: (BuildContext context) => CupertinoAlertDialog(
+                title: const Text('Alert'),
+                content: const Text('This is a sample alert dialog.'),
+                actions: <CupertinoDialogAction>[
```

Comment:
Might be a good idea to add another action with `isDestructiveAction` set to true to show the different styling.